app/app/static/custom/ourscript.js

```diff
@@ -211,13 +227,13 @@ function setTime(id) {
 	} else {
 		clearInterval(timerIntervals[id]);
 	}
-	if (timers[id] == 600) {
+	if (timers[id] >= 600) {
 		document.getElementById(id).className = 'yellowTime';
 	}
-	if (timers[id] == 1200) {
+	if (timers[id] >= 1200) {
 		document.getElementById(id).className = 'orangeTime';
 	}
-	if (timers[id] == 1800) {
+	if (timers[id] >= 1800) {
```

Comment:
this should be of different pull request. A pull request should be tied to the change it was addressing

Reply:
yeah, I was thinking of combining it since it's just a small change. Probably should leave it for that branch for a cleaner GitHub view

Reply:
Your choice... Your pr is good enough... If you want to split it it's your choice

Reply:
Merge whenever you are ready

Reply:
I'll note C50 in the PR name as well, should be good.